pages/instructorQuestion/copyForm.ejs

```diff
@@ -0,0 +1,51 @@
+<form name="copy-question-form" class="needs-validation" method="POST" novalidate>
+  <input type="hidden" name="__action" value="copy_question">
+  <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+  <div class="form-group">
+    <label for="to-course-id-select">The copied question will be added to the following course:</label>
+    <select class="form-control" id="to-course-id-select" name="to_course_id" required>
+      <% if (course.options.isExampleCourse) { %>
+      <option hidden disabled selected value> -- select a course -- </option>
+      <% } %>
+      <% courses.forEach((c) => { %>
```

Comment:
If I'm understanding correctly, `locals.courses` is generated by `middlewares/selectCourses`, which doesn't filter by View/Edit permissions, so we might be showing courses here that the user can't edit.

Reply:
Fixed